lib/rubycritic/cli/options.rb

```diff
@@ -27,6 +27,10 @@ def parse
             @format = format
           end
 
+          opts.on("-s", "--minimum-score [MIN_SCORE]", "Set a minimum score") do |min_score|
+            @minimum_score = min_score.to_f
```

Comment:
Don't we want to use an integer here?
And let's rather use proper constructor methods:

>>
[1] pry(main)> nil.to_i
=> 0
[2] pry(main)> Integer(nil)
TypeError: can't convert nil into Integer
from (pry):2:in `Integer'